share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -132,6 +132,46 @@ macro(rocm_package_add_rocm_core_dependency)
     endif()
 endmacro()
 
+function(parse_python_syspath)
```

Comment:
This function should be prefixed with `rocm_`.

Reply:
sure , will add prefix rocm_